packages/composite-checkout/src/components/checkout-steps.tsx

```diff
@@ -472,11 +473,13 @@ export function CheckoutStepArea( {
 	children,
 	className,
 	submitButtonHeader,
+	submitButtonFooter,
```

Comment:
This is a great addition; could you add it to the README? https://github.com/Automattic/wp-calypso/blob/cae8708b131e1790f016a42cf54e50be065fc556/packages/composite-checkout/README.md#L222